Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php

```diff
@@ -30,6 +30,74 @@ public function provideFixCases()
     {
         return array(
             array(
+                '$array = array(
+                "closure" => function ($param1, $param2) {
+                    return;
+                }
+            );', ),
+            array(
+                '<?php
+        return new JsonResponse(array(
+            "result" => "OK",
+            "html"   => 1, array(
+                "foo"    => "bar",
+                "foofoo" => array(
+                    "a"  => 1,
+                    "b"  => 2
+                )
+            ),)
+        );',
+                '<?php
+        return new JsonResponse(array(
+            "result" => "OK",
+            "html" => 1, array(
+                "foo" => "bar",
+                "foofoo" => array(
+                    "a" => 1,
+                    "b"  =>  2
+                )
+            ),)
+        );',
+            ),
+            array(
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html"   => renderView("views/my_view.html.twig", array(
+                "foo"    => "bar",
+                "foofoo" => 43,
+            )),
+        ]);',
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html" =>    renderView("views/my_view.html.twig", array(
+                "foo" => "bar",
+                "foofoo" => 43,
+            )),
+        ]);',
+            ),
+            array(
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html"   => renderView("views/my_view.html.twig", [
+                "foo"    => "bar",
+                "foofoo" => 42,
+            ]),
+            "baz" => "OK",
```

Comment:
hmm, maybe I misunderstand `consecutive` here.
For example:
```
<?php
    $data1 = [
        "foo"   => "Bar",
        "main1" => [array("baz" => "Test")],
        "bar"   => array(),
    ];

   // couple of 100s lines of code

    $data2 = [
        "foo"   => "Bar",
        "main2" => [array("bazaaaaaaaaaaaaaaaaaaaaaaaaa" => "Test")],
        "bar"   => array(),
    ];
```

do you really want to align data 1[ main1 ] with data 2[ main2 ] ?